source/change-streams/tests/README.rst

```diff
@@ -158,4 +158,41 @@ The following tests have not yet been automated, but MUST still be tested
 #. The ``killCursors`` command sent during the "Resume Process" must not be allowed to throw an exception.
 #. ``$changeStream`` stage for ``ChangeStream`` against a server ``>=4.0`` that has not received any results yet MUST include a ``startAtOperationTime`` option when resuming a changestream.
 #. ``ChangeStream`` will resume after a ``killCursors`` command is issued for its child cursor.
-
+#. ``ChangeStream`` against a server ``>=4.2`` must track the latest ``postBatchResumeToken`` included in an ``aggregate`` or ``getMore`` response.
+#. ``ChangeStream`` must resume ``resumeAfter`` value that always agrees with the value which was specified in the ``resumeAfter`` option while executing an ``aggregate`` command.
+#. - For a ``ChangeStream`` under these conditions:
+      - running against a server ``>=4.2``.
+      - the batch is empty or has been iterated to the last document.
+   - Expected result: 
+       - The ``getResumeToken`` must return the ``postBatchResumeToken`` from the current command response.
+#. - For a ``ChangeStream`` under these conditions:
+      - running against a server ``<4.2``.
+      - the batch is empty or has been iterated to the last document.
+   - Expected result: 
+       - The ``getResumeToken`` must return the ``startAfter`` or ``resumeAfter`` from the current command request. 
```

Comment:
Suggest changing "from the current command request" to  "from the initial aggregate"